rules/windows/persistence_powersshell_profiles.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2022/10/13"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/10/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation or modification of a PowerShell profile. PowerShell profile is a script that is executed when
+PowerShell starts to customize the user environment, which can be abused by attackers to persist in a environment where
+PowerShell is common.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Persistence via PowerShell profile"
+references = ["https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_profiles"]
+risk_score = 47
+rule_id = "5cf6397e-eb91-4f31-8951-9f0eaa755a31"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+  file.path : ("C:\\Users\\*\\Documents\\WindowsPowerShell\\*",
```

Comment:
![image](https://user-images.githubusercontent.com/26856693/195693398-be053fb6-62cf-42ec-93ac-2a926162ee8d.png)

I've tried to reproduce the example in the reference link but it only worked in `Documents\WindowsPowerShell\` instead of `Documents\PowerShell\`

Reply:
maybe `Documents\PowerShell\` apply for older windows versions, you can add it as a precaution